db/schema.rb

```diff
@@ -63,6 +63,16 @@
 
   add_index "date_ranges", ["event_id"], name: "index_date_ranges_on_event_id"
 
+  create_table "emails", force: :cascade do |t|
+    t.boolean  "hide_recipients"
+    t.string   "recipients"
+    t.string   "reply_to"
+    t.string   "subject"
+    t.string   "content"
+    t.datetime "created_at",      null: false
+    t.datetime "updated_at",      null: false
+  end
+
```

Comment:
I think we added this when we tried something with emails (like email knows event and can thus lock it).
Anyway, since we now do something different, we definitely don't need this anymore. (The email model should probably be enough, but we can talk about this again when we have an issue concerning emails)